evennia/commands/default/building.py

```diff
@@ -1558,7 +1562,7 @@ class CmdTypeclass(COMMAND_DEFAULT_CLASS):
     set or change an object's typeclass
 
     Usage:
-      @typclass[/switch] <object> [= <typeclass.path>]
+      @typeclass[/switch] <object> [= typeclass_path]
```

Comment:
Thanks, this was to make it consistent with the doc in line 484 (CmdCreate usage string).  Want me to make them both 'typeclass.path'?